sections/05-adc-data-publishing.qmd

```diff
@@ -2,6 +2,8 @@
 title: "Documenting and Publishing Data"
 ---
 
+TODO: UPDATE ATTRIBUTE-WALKTHROUGH IMAGES TO REFLECT A NETCDF FILE OR SIMILAR
```

Comment:
I don't think this is really necessary. If I were you I would get the ones from this version: https://learning.nceas.ucsb.edu/2022-02-arctic/documenting-and-publishing-data.html#publishing-data-from-the-web

Reply:
I already have those in there except for I think 1 image, I'll switch it out. Thought netCDF might be better but i'm happy to just use the other images